webcommon/libs.graaljs/manifest.mf

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 AutoUpdate-Show-In-Client: false
-OpenIDE-Module: org.netbeans.libs.graaljs/1
+OpenIDE-Module: org.netbeans.libs.graaljs/2
```

Comment:
Why new major version ?

Reply:
There were incompatible changes in the `IS_MODULE_BODY` & co. values.